protected/controllers/DayOneController.php

```diff
@@ -21,78 +21,81 @@ public function actionError()
 
     public function actionNumber()
     {
-        $count_print = array();
+        $displayCount = [];
+        $maxCount     = 10;

```

Comment:
TIP: in SDD, we usually move literals to constants on a class where it belongs. 

```
const MAX_COUNT = 10;
```

* TIP - helpful information in future coding
* PREROG - code improvement suggestion that the dev may / may not implement
* _no label_ - required